### PR TITLE
[iOS] Update README.md

### PR DESCRIPTION
**Pull request by coffmark**
## Issue
- none

## Overview (Required)
- Update README.md

The first time I cloned it, the build failed.
It's listed in the Workaround section, but I think it needed to be in the GetStarted section because it's important information.
If you don't mind, I would appreciate it if you could merge it in.
Otherwise, you can close it. 🙇‍♂️

Thanks!

## Links
- [Preview README.md](https://github.com/DroidKaigi/conference-app-2022/commit/c0710b2d1d0ae37194df63a19f7857cde2da7855?short_path=75c65e6#diff-75c65e64b1a8ab0565021c4f028830763b5ae59437c4f87653301458a81135b5)

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Review by ry-itto — COMMENTED** [review]
Good catch!

However, if it is written there, duplicate information will be appeared.
So, how about add that information to next line of `Xcode 13.4.1` ? 👀 

Example:

## Requirements
- Xcode 13.4.1

  :warning: If you cloned for the first time, you need to try some workaround. Please check it. [Workaround needed for build](#workaround)

- Xcode 14.0 or Later (Recommended)

**Comment by coffmark** [comment]
That's Good! Fixed! https://github.com/DroidKaigi/conference-app-2022/pull/453/commits/22ffd7b9e6d8989e76ce6c647acf501fba729b29

Thanks for your suggestions!

File: app-ios/README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ cd ../
 ```
 
 ## Requirements
-- Xcode 13.4.1 ([Workaround needed for build](#workaround))
+- Xcode 13.4.1
+⚠️ If you cloned for the first time, you need to try some workaround. Please check it. [Workaround needed for build](#workaround)
```
Comment (ry-itto) [inline app-ios/README.md:15]: I remember GitHub has a special syntax for markdown.
How about apply this? 👀 

```suggestion
  > **Warning**
  > If you cloned for the first time, you need to try some workaround. Please check it. [Workaround needed for build](#workaround)
```

https://github.com/community/community/discussions/16925

Comment (coffmark) [reply app-ios/README.md]: Thanks!

**Review by ry-itto — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**